app/views/content/is-teaching-right-for-me/teacher-pay-and-benefits.md

```diff
@@ -4,9 +4,10 @@ heading: "Teacher pay and benefits"
 description: |-
   All qualified teachers will have a starting salary of at least £30,000 (or higher in London). Find out about teacher pay ranges and more benefits of teaching.
 date: "2021-06-24"
-image: "static/content/hero-images/0008.jpg"
 backlink: "../../"
-navigation: 25
+navigation: 5.10
+navigation_title: How much do teachers get paid?
+navigation_description: Find out how much teachers earn and the different ranges of pay available.
```

Comment:
I'm not sure this sounds quite right - almost implies they can choose their pay range?

Should we be talking about starting salary and something about how this progresses/increases etc.?